sphinxdocs/src/sphinx_bzl/bzl.py

```diff
@@ -1766,6 +1766,11 @@ def _on_missing_reference(app, env: environment.BuildEnvironment, node, contnode
     # There's no Bazel docs for None, so prevent missing xrefs warning
     if node["reftarget"] == "None":
         return contnode
+
+    # Any and object are just conventions from Python, but useful for
+    # indicating what something is in Starlark, so treat them specially.
+    if node["reftarget"] in ("Any", "object"):
```

Comment:
`node["reftarget"].lower() in ("any", "object"):`? I'm not sure if `any` is different from `Any`.

Reply:
In python land, it's case sensitive. This is all informal, so we can kind of make it up. And for doc purposes, either might be convenient. For now, I'm just gonna match Python.